.containerignore

```diff
@@ -0,0 +1,16 @@
+.ansible-test
```

Comment:
Maybe symlink this against `.gitignore`?

Reply:
If i remember correctly they are not fully compatible.

Reply:
I don't remember any obvious incompatibilities but to the extent of things used here, it probably doesn't matter but it'd be easier to maintain the copies of exclusions.